src/foam/dao/MDAO.java

```diff
@@ -159,7 +159,7 @@ public Sink select_(X x, Sink sink, long skip, long limit, Comparator order, Pre
     // TODO: if plan cost is >= size, log a warning
     if ( state != null && predicate != null && plan.cost() > 1000 && plan.cost() >= index_.size(state) ) {
       Logger logger = (Logger) x.get("logger");
-      logger.error(predicate.createStatement(), " Unindexed search on MDAO");
```

Comment:
I'm trying to think if there might be value to log the predicate that the user actually requested along with the one that we partialEval'd to.  Might be useful to log both.

Reply:
Added one in 3b1b1cf